### DISCUSSION TITLE
intput can  not recognize

### DISCUSSION THREAD

**Discussion by junayedahamed** [mozaddedalfeshani/miniCppProject#4] [Q&A]
if i give a character in int section (phone number section ) then program running unlimited time

**Comment by mozaddedalfeshani** [comment]
Thank's for your kind information I will fix it as soon as possible

**Comment by junayedahamed** [comment]
@mozaddedalfeshani ok

**Comment by mozaddedalfeshani** [comment]
@junayedahamed  Dear I am solved room and phone number entering problem! If you found more problem please notify as soon as possible

**Reply by junayedahamed** [reply]
@mozaddedalfeshani ok i will inform you if i get any more proble after checking that

**Comment by mozaddedalfeshani** [comment]
@junayedahamed Could you please review my code and where I should need to edit that I can understand about my code ! How my code run on different os ! 👍
